app/javascript/lib/validation/validate-remote-file-content.js

```diff
@@ -0,0 +1,71 @@
+/**
+* @fileoverview Client-side file validation (CSFV) for sync
+*/
+
+import { validateFileContent } from './validate-file-content'
+import { fetchBucketFile } from 'lib/scp-api'
+
+/**
+ * 50 MiB, max number of bytes to read and validate from remote file for sync.
+ * Median fixed US bandwidth is 16 MiB/s (17 MB/s, 136 Mbps) as of 2021-12
+ * per https://www.speedtest.net/global-index/united-states.  > 80% of ingests
+ * are for files <= 50 MiB per https://mixpanel.com/s/3EbFMj.
+ *
+ * So 50 MiB means sync CSFV fully scans > 80% files, usually in < 5 seconds.
+ * Local tests gave 4.6 s (3.4 s remote read, 1.1 s validate; 138 Mbps down).
+ */
+export const MAX_SYNC_CSFV_BYTES = 50 * 1024 * 1024
```

Comment:
My understanding of `maxBytesPerLine` is that setting the default at 1 GiB ensures that very long lines can be _read in from memory_, while giving some protection that pathologically long lines don't freeze the UI.  (@devonbush, please correct me if that's wrong!)

`MAX_SYNC_CSFV_BYTES`, on the other hand, is a crude way to help ensure we _don't spend too much time downloading from network`_.  This value is much lower simply because network reads are so much slower than memory reads.

Reply:
That's exactly right -- the chunker has to hold all of a single line in memory at once, even though the line may be split into chunks.  Having a max line prevents the browser from running out of RAM trying to read a file that doesn't have newlines